### PR TITLE
Updated RoseHub Project description and Added more Projects

### PR DESCRIPTION
**Pull request by chefallan**

### CODE REVIEW

**Comment by chefallan** [comment]
Sorry i forgot to push the project description update of RoseHub immediately .

**Comment by chefallan** [comment]
@warengonzaga kindly check it, thanks.

**Comment by warengonzaga** [comment]
Checking @itsnalla

**Comment by chefallan** [comment]
Added more projects @kquirapas @warengonzaga .

File: json/data.json [hunk 1]
```diff
@@ -703,7 +703,7 @@
     "name": "RoseHub",
```
Comment (kquirapas) [inline json/data.json:703]: LGTM

**Review by kquirapas — APPROVED** [review]

**Pull request merged**